private/dev/Dashboard.js

```diff
@@ -87,7 +87,7 @@ export default () => {
     // .use(Unsplash, { target: Dashboard, companionUrl: COMPANION_URL, companionAllowedHosts })
     .use(RemoteSources, {
       companionUrl: COMPANION_URL,
-      sources: ['Box', 'Dropbox', 'Facebook', 'GoogleDrive', 'Instagram', 'OneDrive', 'Unsplash', 'Url'],
+      sources: ['Box', 'Dropbox', 'Facebook', 'GoogleDrive', 'Instagram', 'OneDrive', 'Unsplash', 'Zoom', 'Url'],
```

Comment:
Isn't  that be causing problems if devs don't have Zoom credentials in their `.env`?

Reply:
Not as long as you don't try to use the Zoom plugin right?

Reply:
Yup, I use some of the others also without credentials, and it only fails once you try to use those providers